### PR TITLE
Add Polish localized content of the main page

### PR DESCRIPTION
**Pull request by mfilocha**
This is a main page of kubernetes.io translated to Polish.

### CODE REVIEW

File: content/pl/_index.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+---
+{{< announcement >}}
+
+{{< deprecationwarning >}}
+
+{{< blocks/section id="oceanNodes" >}}
+{{% blocks/feature image="flower" %}}
+### [Kubernetes (K8s)]({{< relref "/docs/concepts/overview/what-is-kubernetes" >}}) to otwarte oprogramowanie służące do automatyzacji procesów uruchamiania, skalowania i zarządzania aplikacjami w kontenerach.
+
+Kubernetes grupuje kontenery, które są składowymi jednej aplikacji, w logicznie grupy, ułatwiając zarządzanie nimi i ich odnajdywanie. Kubernetes korzysta z [piętnastoletniego doświadczenia Google w uruchamianiu wielkoskalowych serwisów](http://queue.acm.org/detail.cfm?id=2898444) i łączy je z najlepszymi pomysłami i praktykami wypracowanymi przez społeczność.
```
Comment (kpucynski) [inline content/pl/_index.html]:
```suggestion
Kubernetes grupuje kontenery, które są częścią jednej aplikacji, w logicznie grupy, ułatwiając ich odnajdywanie i zarządzanie nimi. Korzysta z [piętnastoletniego doświadczenia Google w uruchamianiu wielkoskalowych serwisów](http://queue.acm.org/detail.cfm?id=2898444) i łączy je z najlepszymi pomysłami i praktykami wypracowanymi przez społeczność.
```

File: content/pl/_index.html [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+{{% blocks/feature image="scalable" %}}
+#### W skali globalnej
+
+Zaprojektowany według tych samych zasad, które pozwalają Google uruchamiać miliardy kontenerów każdego tygodnia, Kubernetes może skalować się bez konieczności powiększania zespołu administratorów.
+
+{{% /blocks/feature %}}
+
+{{% blocks/feature image="blocks" %}}
+#### Nigdy nie za mały
```
Comment (kpucynski) [inline content/pl/_index.html]:
```suggestion
#### Nigdy za mały

File: content/pl/_index.html [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+Niezależnie, czy prowadzisz tylko testy, czy globalny koncern, dzięki elastyczności Kubernetesa Twoje aplikacje mogą być instalowane w łatwy i nieprzerwany sposób, bez względu na to, jak bardzo skomplikowane są Twoje wymagania.
+
+{{% /blocks/feature %}}
+
+{{% blocks/feature image="suitcase" %}}
+#### Działa w każdym środowisku
+
+Kubernetes, projekt open-source, daje Ci wolność wyboru ⏤ skorzystaj z prywatnego centrum danych, infrastruktury hybrydowej czy chmury publicznej. Bez wysiłku możesz przenieść swoje aplikacje tam, gdzie są najbardziej potrzebne.
```
Comment (kpucynski) [inline content/pl/_index.html]:
```suggestion
Kubernetes jako projekt open-source daje Ci wolność wyboru ⏤ skorzystaj z prywatnego centrum danych, infrastruktury hybrydowej lub chmury publicznej. Bez wysiłku możesz przenieść swoje aplikacje tam, gdzie są najbardziej potrzebne.
```

File: content/pl/_index.html [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+Kubernetes jako projekt open-source daje Ci wolność wyboru ⏤ skorzystaj z prywatnego centrum danych, infrastruktury hybrydowej lub chmury publicznej. Bez wysiłku możesz przenieść swoje aplikacje tam, gdzie są najbardziej potrzebne.
+
+{{% /blocks/feature %}}
+
+{{< /blocks/section >}}
+
+{{< blocks/section id="video" background-image="kub_video_banner_homepage" >}}
+<div class="light-text">
+<h2>The Challenges of Migrating 150+ Microservices to Kubernetes</h2>
+        <p>By Sarah Wells, Technical Director for Operations and Reliability, Financial Times</p>
```
Comment (sftim) [inline content/pl/_index.html]: Consider translating “By”

**Comment by sftim** [comment]
This one, the Polish localization team get to /approve!

File: content/pl/_index.html (hunk 4 above)
Comment (kpucynski) [reply content/pl/_index.html]: hm, in most cases in polish we do not add "by" in bibliography. I would just remove it.
```suggestion
        <p>Sarah Wells, Technical Director for Operations and Reliability, Financial Times</p>
```

Comment (mfilocha) [reply content/pl/_index.html]: Yes, I've just fixed it locally.

**Comment by sftim** [comment]
See #18680

**Pull request merged**